app/models/course_enrichment.rb

```diff
@@ -61,6 +61,16 @@ def is_fee_based?
     course.is_fee_based?
   end
 
+  def duration
+    if course_length == "OneYear"
+      "1 year"
+    elsif course_length == "TwoYears"
+      "Up to 2 years"
+    else
+      course_length
+    end
+  end
```

Comment:
I just had a chat with Jake about what this is, and I think this should be in the serializer until we sort out what we're doing with this field, exactly. Nothing else is going using `duration` at this point, and it would be unfortunate it something else decided to re-use it, I think.

Reply:
Moved duration to serializer